sys/include/net/gnrc/mac/types.h

```diff
@@ -40,12 +37,151 @@ extern "C" {
  * @brief definition for device transmission feedback types
  */
 typedef enum {
-    TX_FEEDBACK_UNDEF = 0,    /* Transmission just start, no Tx feedback yet */
-    TX_FEEDBACK_SUCCESS,      /* Transmission succeeded */
-    TX_FEEDBACK_NOACK,        /* No ACK for the transmitted packet */
-    TX_FEEDBACK_BUSY          /* found medium busy when doing transmission */
+    TX_FEEDBACK_UNDEF = 0,    /**< Transmission just start, no Tx feedback yet */
+    TX_FEEDBACK_SUCCESS,      /**< Transmission succeeded */
+    TX_FEEDBACK_NOACK,        /**< No ACK for the transmitted packet */
+    TX_FEEDBACK_BUSY          /**< found medium busy when doing transmission */
 } gnrc_mac_tx_feedback_t;
 
+/**
+ * @brief Static initializer for gnrc_mac_tx_feedback_t.
+ */
+#define GNRC_MAC_TX_FEEDBACK_INIT { TX_FEEDBACK_UNDEF }
+
+#if ((GNRC_MAC_RX_QUEUE_SIZE != 0) || (GNRC_MAC_DISPATCH_BUFFER_SIZE != 0))
```

Comment:
@miri64 By the way, by adding `|| defined(DOXYGEN)`, won't that change the original `#if` logic? Since, if `DOXYGEN` is defined, then `#if` will always be ture?